### PR TITLE
feat: added page-grid module (alpha)

### PR DESCRIPTION
**Pull request by ianmcburnie**
Systemizes the page-grid system currently being used on the skin website. The grid system is still in early development (and contains minimal "features") and so the documentation will remain hidden for now.

### CODE REVIEW

**Review by austingardner — APPROVED** [review]

File: src/less/page-grid/base/page-grid.less [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.page-grid-container {
+    display: flex;
+    justify-content: center;
+    margin: 0 @spacing-small;
+}
```
Comment (agliga) [inline src/less/page-grid/base/page-grid.less:5]: Is this needed? Can we not put this in the page grid class? I think we can just do `justify-content: center` there and it should work I assume? https://css-tricks.com/snippets/css/complete-guide-grid/#justify-content

**Review by agliga — COMMENTED** [review]
Also are docs files supposed to be changed?

File: src/less/page-grid/base/page-grid.less (hunk 1 above)
Comment (ianmcburnie) [reply src/less/page-grid/base/page-grid.less:5]: It needs to be a flex item/child for that to work though, right? 

Try here: https://ebay.github.io/skin/?debug=1

Comment (agliga) [reply src/less/page-grid/base/page-grid.less:5]: I think we can do something with margin auto, either way... I think it's fine as is.

**Review by agliga — APPROVED** [review]

File: src/less/page-grid/base/page-grid.less (hunk 1 above)
Comment (ianmcburnie) [reply src/less/page-grid/base/page-grid.less:5]: `margin: 0 auto` would center the grid element... but then we lose the ability to add the outer margins.

Comment (ianmcburnie) [reply src/less/page-grid/base/page-grid.less:5]: Unless there's something clever we can do with min, max, minmax etc for the horizonatal value...

Comment (austingardner) [reply src/less/page-grid/base/page-grid.less:5]: I think we could try something like 
```css
.page-grid-container {
  display: grid;
  place-items: center;
}
```

I think that would center things correctly, wouldn't it?

Comment (ianmcburnie) [reply src/less/page-grid/base/page-grid.less:5]: Yeah, the good thing about having a container is that it gives us this level of flexibility if we want it.

Comment (ianmcburnie) [reply src/less/page-grid/base/page-grid.less:5]: I'm going to merge this. We can always refine as we go.

**Pull request merged**